src/Microsoft.Extensions.Configuration/MemoryConfigurationBuilderExtensions.cs

```diff
@@ -7,6 +7,9 @@
 
 namespace Microsoft.Extensions.Configuration
 {
+    /// <summary>
+    /// Adds extension methods for the MemoryConfigurationProvider.
```

Comment:
Oh, and plus, that's not what this is 😄 It's extension methods for the builders, not providers...